dhis-2/dhis-services/dhis-service-tracker/src/test/java/org/hisp/dhis/tracker/validation/hooks/EventDataValuesValidationHookTest.java

```diff
@@ -693,15 +703,14 @@ void failValidationWhenDataElementValueTypeIsInvalid()
         runAndAssertValidationForDataValue( ValueType.DATETIME, "wrong_date_time" );
         runAndAssertValidationForDataValue( ValueType.COORDINATE, "10" );
         runAndAssertValidationForDataValue( ValueType.URL, "not_valid_url" );
-        runAndAssertValidationForDataValue( ValueType.FILE_RESOURCE, "not_valid_uid" );
     }
 
     @Test
     void successValidationDataElementOptionValueIsValid()
     {
         setUpIdentifiers();
 
-        DataValue validDataValue = dataValue( "code" );
+        DataValue validDataValue = dataValue( "CODE" );
```

Comment:
why does that matter? is this value always uppercase?

Reply:
The title of the PR is refactor, so I would not expect test data or expectations to change since behavior should be preserved in a refactoring. That is why I am asking 😋 So previously `code` was deemed valid which it should not have been since the value has to match the metadata exactly?